docs/services/motion/_index.md

```diff
@@ -103,7 +104,7 @@ By default, motion is unconstrained with the exception of obstacle avoidance.
 For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/motion/index.html#viam.services.motion.MotionClient.move).
 
 ```python {class="line-numbers linkable-line-numbers"}
-motion = MotionClient.from_robot(robot=robot, name="builtin")
+motion = MotionClient.from_robot(robot=robot, name="my_motion_service")
```

Comment:
I think you don't have to configure the motion service and you just use the builtin service which is why this is and should be `builtin` is that wrong?

Reply:
Ohhh you must be right, it does say you don't configure it! Thanks for catching. I think it might be beneficial to add a note to clarify that on this page, so I will make a ticket for that? @npentrel

Reply:
sure or just fix it in this PR :)